### PR TITLE
Dvory/middleware

### PR DESCRIPTION
**Pull request by mindas2000**

### CODE REVIEW

**Review by est8906 — APPROVED** [review]
Very readable and clear code
I did not find empty lines, console log, etc

File: server-side/.env [hunk 1]
```diff
@@ -1,5 +1,5 @@
-PORT=3000
+PORT=5000
 password=EuACQK5n92JI3vn4
-DB_URL="mongodb://127.0.0.1:27017/TimeOut"
+DB_URL='mongodb+srv://sh3000444:EuACQK5n92JI3vn4@cluster0.2ttcjcg.mongodb.net/?retryWrites=true&w=majority&appName=Cluster0'
```
Comment (SariGlick) [inline server-side/.env:3]: remove changing the mongo url
please add server-side/.env & package-lock.json to .gitignore file

File: statistics-service/.env [hunk 2]
```diff
@@ -0,0 +1,3 @@
+
+PORT=5000
```
Comment (SariGlick) [inline statistics-service/.env:2]: remove this file

File: statistics-service/.gitignore [hunk 3]
```diff
@@ -0,0 +1,2 @@
+/node_modules
```
Comment (SariGlick) [inline statistics-service/.gitignore:1]: remove this file

File: client-side/src/axios/middleware.js [hunk 4]
```diff
@@ -0,0 +1,29 @@
+import axios from 'axios';
+import dotenv from 'dotenv'
+
+dotenv.config()
```
Comment (SariGlick) [inline client-side/src/axios/middleware.js:4]: please add ; to this line and the next one too

File: server-side/controllers/profile.controller.js [hunk 5]
```diff
@@ -4,18 +4,13 @@ import Profiles from '../models/profile.model.js';
 
 export const getAllProfiles = async (req, res, next) => {
     try {
-        const profiles = await Profiles.find().populate('limitedWebsites.websiteId blockedSites').select('-__v');
+        const profiles = await Profiles.find().populate('listWebsites').select('-__v');
```
Comment (SariGlick) [inline server-side/controllers/profile.controller.js:7]: Are the changes in the file related to this task? If not, remove them

**Review by SariGlick — CHANGES_REQUESTED** [review]

File: server-side/.env (hunk 1 above)
Comment (mindas2000) [reply server-side/.env:3]: ✅

File: statistics-service/.env (hunk 2 above)
Comment (mindas2000) [reply statistics-service/.env:2]: ✅

File: statistics-service/.gitignore (hunk 3 above)
Comment (mindas2000) [reply statistics-service/.gitignore:1]: ✅

File: client-side/src/axios/middleware.js (hunk 4 above)
Comment (mindas2000) [reply client-side/src/axios/middleware.js:4]: ✅

File: server-side/controllers/profile.controller.js (hunk 5 above)
Comment (mindas2000) [reply server-side/controllers/profile.controller.js:7]: ✅

**Review by SariGlick — APPROVED** [review]

**Review by SariGlick — APPROVED** [review]

**Pull request merged**